Controller/MediaAdminController.php

```diff
@@ -111,4 +112,26 @@ public function uploadAction()
         ));
     }
 
+    /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     * @param string   $theme
+     */
+    protected function setFormTheme(FormView $formView, $theme)
+    {
+        /** @var \Twig_Environment $twig */
+        $twig = $this->get('twig');
+        if ($twig->hasExtension('Symfony\Bridge\Twig\Extension\FormExtension')){
```

Comment:
IMO we should use the same method as SonataAdminBundle for more compliance: 
* https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Controller/GalleryAdminController.php#L67-L85
* https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Controller/MediaAdminController.php#L111-L129